### PR TITLE
Module-config

### PR DESCRIPTION
**Pull request by BenediktMKuehne**
**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
---
Expert-mode changes
Makes scan modules configurable manually before running the analysis
Adding Template
Add links

**What is the current behavior?** (You can also link to an open issue here)
---
everything runs

**What is the new behavior (if this is a feature change)? If possible add a screenshot.**
---
disable/enable via checkbox
image

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
---
nope
just pull and run

### CODE REVIEW

**Comment by BenediktMKuehne** [comment]
depends on #81

**Comment by BenediktMKuehne** [comment]
image

caused by 
`2023-03-01 10:14:10,786 87081 139908645623360 /var/www/embark/uploader/boundedexecutor.py INFO Success: cd /var/www/emba && sudo ./emba -p ./scan-profiles/default-scan-no-notify.emba -f /var/www/active/29e17c18-6317-40cb-8687-e91ea7f3fff3/DIR300B5_FW214WWB01.bin -l /var/www/emba_logs/29e17c18-6317-40cb-8687-e91ea7f3fff3/emba_logs  -X "b5" -Z "dir300" -Y "dlink" -N "asd (uuid:29e17c18-6317-40cb-8687-e91ea7f3fff3)" -m s40 -m s45 -m s50 -m s90
2023-03-01 10:14:10,786 87081 139908645623360 /var/www/embark/uploader/boundedexecutor.py INFO Success: cd /var/www/emba && sudo ./emba -p ./scan-profiles/default-scan-no-notify.emba -f /var/www/active/29e17c18-6317-40cb-8687-e91ea7f3fff3/DIR300B5_FW214WWB01.bin -l /var/www/emba_logs/29e17c18-6317-40cb-8687-e91ea7f3fff3/emba_logs  -X "b5" -Z "dir300" -Y "dlink" -N "asd (uuid:29e17c18-6317-40cb-8687-e91ea7f3fff3)" -m s40 -m s45 -m s50 -m s90
2023-03-01 10:14:10,786 87081 139908645623360 /var/www/embark/uploader/boundedexecutor.py INFO EMBA returned: 1
2023-03-01 10:14:10,786 87081 139908645623360 /var/www/embark/uploader/boundedexecutor.py INFO EMBA returned: 1
`

@m-1-k-3 are brackets and stuff allowed?

**Comment by BenediktMKuehne** [comment]
depends on 91

File: embark/uploader/forms.py [hunk 1]
```diff
@@ -32,15 +32,63 @@ class Meta:
...
+        ('s99', 'S99_grepit'),
+        ('s100', 'S100_command_inj_check'),
+        ('s106', 'S106_deep_key_search'),
+        ('s107', 'S107_deep_password_search'),
+        ('s108', 'S108_stacs_password_search'),
+        ('s109', 'S109_jtr_local_pw_cracking'),
+        ('s110', 'S110_yara_check'),
+        ('s115', 'S115_usermode_emulator'),
+        ('s116', 'S116_qemu_version_detection'),
+        ('s120', 'S120_cwe_checker')
```
Comment (m-1-k-3) [inline embark/uploader/forms.py:74]: Could we auto-generate this list in the future?

File: run-server.sh [hunk 2]
```diff
@@ -51,6 +51,7 @@ import_helper()
 cleaner() {
   pkill -u root daphne
   pkill -u root "$PWD"/emba/emba
+  pkill -u root "$PWD"/emba/emba
```
Comment (m-1-k-3) [inline run-server.sh]: is it not enough to kill it once ... do not shoot two times the same victim ;)

File: embark/uploader/forms.py (hunk 1 above)
Comment (BenediktMKuehne) [reply embark/uploader/forms.py:74]: possibly

**Review by m-1-k-3 — APPROVED** [review]

**Pull request merged**